tests/spectrum_test.py

```diff
@@ -243,34 +243,37 @@ def test_negative_input(spec_data):
 log_bins = np.logspace(1, 4, num=NEDGES, base=10.0)
 
 
-def make_spec_listmode(t, use_cal=False):
+def make_spec_listmode(t, is_cal=False, apply_cal=False):
     if t == 'uniform':
-        spec = bq.Spectrum.from_listmode(lmd, bins=NBINS, xmin=XMIN, xmax=XMAX)
+        spec = bq.Spectrum.from_listmode(lmd, is_cal=is_cal, bins=NBINS, xmin=XMIN, xmax=XMAX)
     elif t == 'log':
-        spec = bq.Spectrum.from_listmode(lmd, bins=log_bins)
+        spec = bq.Spectrum.from_listmode(lmd, is_cal=is_cal, bins=log_bins)
     elif t == 'default':
-        spec = bq.Spectrum.from_listmode(lmd)
+        spec = bq.Spectrum.from_listmode(lmd, is_cal=is_cal)
     else:
         return t
 
-    if use_cal:
+    if apply_cal:
         cal = bq.LinearEnergyCal.from_coeffs({'m': TEST_GAIN, 'b': 0.0})
```

Comment:
This should be  raising a `DeprecationWarning` -- should we take this as an opportunity to update to the new `Calibration`? Something like:

```python
cal = bq.Calibration.from_polynomial([0.0, TEST_GAIN])
```

Reply:
I don't see the DeprecationWarning because this branch is way behind main. I'll resolve conflicts and check.